api/src/main/java/keywhiz/api/automation/v2/SecretDetailResponseV2.java

```diff
@@ -64,9 +68,21 @@ public Builder secret(Secret secret) {
           .createdAtSeconds(secret.getCreatedAt().toEpochSecond())
           .createdBy(secret.getCreatedBy())
           .type(secret.getType().orElse(null))
+          .expiry(secret.getExpiry())
           .metadata(secret.getMetadata());
     }
 
+    public Builder secretContent(SecretContent secretContent) {
```

Comment:
The name seems slightly confusing given it has no actual secret content, just metadata.